docs/devsite-help/getting-started.md

```diff
@@ -68,17 +68,25 @@ dotnet add package Google.Cloud.Translate.V3
 > - Grpc.Auth, Grpc.Core.Api, Grpc.Net.Client, Grpc.Net.Common: support for the [gRPC](https://grpc.io/) RPC protocol
 > - Google.LongRunning: support for [long-running operations](long-running-operations.md)
 
+## Set up Application Default Credentials
+
+This step sets up Application Default Credentials to use your user credentials in a local development
+environment. 
+
+```sh
+gcloud auth application-default login
+```
+
+When you create a client, it automatically detects and uses these credentials.
+For other ways to set up Application Default Credentials, see
+[Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc).
+
 ## Create a client
 
 The first step in making any API calls is to create a client. Some libraries have multiple clients
 for operations involving different resources; others have a single client. In the Translation API
 we're using, there's just `TranslationServiceClient`.
 
-Clients can be configured in a number of ways, but in many cases the defaults are fine. The most
-common reason to use explicit configuration is to use specific credentials for
-[authentication](https://cloud.google.com/docs/authentication/use-cases). For this example, we'll just use the
-[application default credentials](https://cloud.google.com/docs/authentication/production#automatically).
-
```

Comment:
Similar to @jskeet I'd prefer if we rephased this section instead of removing it and even instead of adding a whole new section on setting up  ADC.

I think something along the lines of: "... For this example, we'll just use ADC (with links to ADC and general auth at Google). To set up ADC in your local development environment [link]".

Note that instead of including the gcloud command verbatim, I'd prefer a link to a document that does.

Reply:
Aditionally, this section is not just talking about specific credentials. It's also mentioning that "clients can be configured in other ways but defaults are mostly fine", and that's relevant information to include here, regardles of ADC usage.